async_upnp_client/cli.py

```diff
@@ -68,11 +68,17 @@
 )
 subparser = subparsers.add_parser("search", help="Search for devices")
 subparser.add_argument("--bind", help="ip, e.g., 192.168.0.10")
+subparser.add_argument(
+    "--target", help="ip, e.g., 192.168.0.10 or FF02::C to request from"
+)
 subparser.add_argument(
     "--service_type", help="service type to search for", default=SSDP_ST_ALL
 )
 subparser = subparsers.add_parser("advertisements", help="Listen for advertisements")
 subparser.add_argument("--bind", help="ip, e.g., 192.168.0.10")
+subparser.add_argument(
+    "--target", help="ip, e.g., 239.255.255.250 or FF02::C to listen to"
```

Comment:
On line 72 it says 'to request from', but here it says 'to listen to', but both options are named `target`. Is this right?

Reply:
I think so. In one case, we send a message out on the link, in the other case we listen for others to have sent out data to that target multicast address.